Hackerank/Digit Power Sum.py

```diff
@@ -0,0 +1,33 @@
+# Problem Link: https://www.hackerrank.com/contests/projecteuler/challenges/euler119/problem
```

Comment:
This is the problem link  for Project Euler -119 : Digit Power Sum. I have added the code in main file too.